docs/about/compatibility/openmp.md

```diff
@@ -228,14 +228,13 @@ When Xnack support is not needed:
 
 #### Unified shared memory pragma
 
-This OpenMP pragma is available on MI200 through `xnack+` support.
+There are two ways in which you can trigger USM:
 
-```bash
-omp requires unified_shared_memory
-```
+- Add ``#pragma omp requires unified_shared_memory`` in your source files or translation units to be more precise.
```

Comment:
I'd remove "or translation units to be more precise". This is a compiler detail that most users won't find useful.

Reply:
Ya..I had felt the same. Thanks for confirming.